product/reports/170_Configuration Management - Containers/040_Recently Discovered Container Groups.yaml

```diff
@@ -17,7 +17,11 @@ headers:
 - Name
 - Ready Status
 - Creation
-conditions:
+conditions: !ruby/object:MiqExpression
+  exp:
+    IS NOT NULL:
+      field: ContainerGroup.deleted_on
```

Comment:
this is working?
the syntax is defined in `MiqExpression::Field::REGEX` (just fyi it is not needed to read the regex :) )
for this case it should be `ContainerGroup-deleted_on`, the "." is for relations(`ContainerGroup.ext_management_system-name`).
`MiqExpression::Field.parse('ContainerGroup-deleted_on')` => true

`MiqExpression::Field.parse('ContainerGroup.deleted_on')` => nil

so please use 'ContainerGroup-deleted_on' (and also in other places)